src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -575,52 +575,6 @@ suite('Kernel Execution @kernelCore', function () {
             )
         ]);
     });
-    test.skip('Messages from background threads can come in other cell output', async function () {
```

Comment:
Messages will now appear in the right cell, hence the old (broken) behavior no longer applies